src/routes/index.js

```diff
@@ -69,6 +71,13 @@ export default function Routes() {
       <Route path="/faq" exact render={() => <Website component={<FAQ />} />} />
       <Route path="/login" exact render={() => <WebApp component={<Login />} />} />
       <Route path="/dashboard" exact render={() => <WebApp component={<DashboardController />} />} />
+      {/* Tab for a Cause Routes */}
+      {/* Tried below with URLSearchParams, could not get it to work */}
+      {/* <Route path="/tab" exact render={() => <Website component={<Home collab={'true'}/>} />} /> */}
+      <Route path="/tab/girlfriend" exact render={() => <Website component={<Home collab={'Girlfriend Collective'}/>} />} />
+      <Route path="/tab/byhumankind" exact render={() => <Website component={<Home collab={'by Humankind'}/>} />} />
+      <Route path="/tab/cicada" exact render={() => <Website component={<Home collab={'Cicada'}/>} />} />
+      <Route path="/tab/rocketbook" exact render={() => <Website component={<Home collab={'Rocketbook'}/>} />} />
```

Comment:
If you want to use these paths instead of query parameters but also don't want to have to add each one manually, you could try adding an array of paths all at once that you fetch from the Airtable! You can see how we created retailerPaths at the top of this file and then added the whole group of {retailerPaths} on like 81, which helps automate this process :)

Reply:
Cool suggestion! Will make this change when making the Airtable change referenced in the first comment